src/hotspot/share/opto/predicates.cpp

```diff
@@ -32,29 +32,29 @@
 // (i.e. not belonging to an Initialized Assertion Predicate anymore)
 Node* AssertionPredicatesWithHalt::find_entry(Node* start_proj) {
   Node* entry = start_proj;
-  while (is_assertion_predicate_success_proj(entry)) {
+  while (AssertionPredicateWithHalt::is_predicate(entry)) {
```

Comment:
Is it `AssertionPredicateWithHalt::is_predicate` instead of simply `is_predicate` on purpose?